File: RukaLjubavi.API/RukaLjubavi.API/Database/Donator.cs
using System;
using System.Collections.Generic;

namespace RukaLjubavi.API.Database
{
    public partial class Donator
    {
        public Donator()
        {
            Donacija = new HashSet<Donacija>();
            DonatorKategorije = new HashSet<DonatorKategorije>();
        }

        public int Id { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string Jmbg { get; set; }
        public DateTime? DatumRodjenja { get; set; }
        public string MjestoRodjenja { get; set; }
        public string Email { get; set; }
        public string LozinkaSalt { get; set; }
        public string LozinkaHash { get; set; }
        public string Telefon { get; set; }
        public string Adresa { get; set; }
        public bool? IsVerifikovan { get; set; }
        public int? GradId { get; set; }

        public virtual Grad Grad { get; set; }
        public virtual ICollection<Donacija> Donacija { get; set; }
        public virtual ICollection<DonatorKategorije> DonatorKategorije { get; set; }
    }
}

File: RukaLjubavi.Android/app/src/main/java/com/fit/ba/rukaljubavi/Models/Donator.kt
package com.fit.ba.rukaljubavi.Models

import java.io.Serializable

class Donator(var korisnikId: Int ,var ime: String, var prezime: String,var jmbg: String,var email:String,var telefon:String, var adresa: String, var datumRegistracije: String, var isVerifikovan: Boolean, var mjestoPrebivalista: String,var mjestoPrebivalistaId:Int,
              var brojDonacija: Int, var ocjenaPovjerljivost: Float, var ocjenaBrzinaDostavljanja: Float, var ocjenaPostivanjeDogovora: Float)
    : Serializable {
}